### PR TITLE
Updated all Telegram component files configuration variable

### PR DESCRIPTION
**Pull request by klaasnicolaas**
Updated style of all Telegram documentation to follow new configuration variables description.
Related to #6385.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Liquid Exception: (<unknown>): mapping values are not allowed in this context at line 11 column 75 in /opt/build/repo/source/_components/telegram_bot.polling.markdown

do not understand that he gives this error 🤔

File: source/_components/telegram_bot.broadcast.markdown [hunk 1]
```diff
@@ -27,13 +27,28 @@ telegram_bot:
...
+allowed_chat_ids:
+  description: A list of user in the `user_id` Telegram format enabled to interact to webhook
+  required: true
+  type: list
+api_key:
+  description: The API token of your bot.
+  required: true
+  type: string
+parse_mode:
+  description: Default parser for messages if not explicit in message data: 'html' or 'markdown'.
```
Comment (WofWca) [inline source/_components/telegram_bot.broadcast.markdown]: It counts lines relative to the start of the variables section declaration.
There is a `:` with a space following it in the `description`. Quote the string.

File: source/_components/telegram_bot.polling.markdown [hunk 2]
```diff
@@ -29,13 +29,28 @@ telegram_bot:
...
+allowed_chat_ids:
+  description: A list of user in the `user_id` Telegram format enabled to interact to webhook
+  required: true
+  type: list
+api_key:
+  description: The API token of your bot.
+  required: true
+  type: string
+parse_mode:
+  description: Default parser for messages if not explicit in message data: 'html' or 'markdown'.
```
Comment (WofWca) [inline source/_components/telegram_bot.polling.markdown]: Same

File: source/_components/telegram_bot.webhooks.markdown [hunk 3]
```diff
@@ -34,15 +34,38 @@ telegram_bot:
...
+  description: The API token of your bot.
+  required: true
+  type: string
+trusted_networks:
+  description: Telegram server access ACL as list.
+  required: false
+  default: 149.154.167.197-233
+  type: string
+parse_mode:
+  description: Default parser for messages if not explicit in message data: 'html' or 'markdown'.
```
Comment (WofWca) [inline source/_components/telegram_bot.webhooks.markdown]: Same

**Comment by klaasnicolaas** [comment]
I added "" as you described 😉

**Comment by klaasnicolaas** [comment]
> Liquid Exception: (): mapping values are not allowed in this context at line 11 column 75 in /opt/build/repo/source/_components/telegram_bot.polling.markdown
> 
> do not understand that he gives this error 🤔

I still get this error message

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @klaasnicolaas!

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**